presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,54 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Fix a null pointer exception during the stats collection phase in query planning.
```

Comment:
```suggestion
* Improve reliability during the stats collection phase in query planning.
```

Reply:
I like the original better.  
1. our release note guidelines say fixes should begin with the word "fix" (i think we should only stick to that when it's reasonable, but it seems reasonable here).
2. "reliability" is so vague.  Mentioning the exception makes it easier to know whether this release fixed the issue you were seeing.

Reply:
That's fair, I'm OK with anything that doesn't reference Java exceptions.  How about

`Fix error during stats collection ...`